src/libraries/System.Private.CoreLib/src/System/Runtime/Intrinsics/Arm/AdvSimd.cs

```diff
@@ -3744,6 +3744,56 @@ public new abstract class Arm64 : ArmBase.Arm64
             /// </summary>
             public static Vector128<sbyte> VectorTableLookupExtension(Vector128<sbyte> defaultValues, (Vector128<sbyte>, Vector128<sbyte>, Vector128<sbyte>, Vector128<sbyte>) table, Vector128<sbyte> byteIndexes) => VectorTableLookupExtension(defaultValues, table, byteIndexes);
 
+            /// <summary>
+            ///   A64: ST2 { Vn.16B, Vn+1.16B }, [Xn]
+            /// </summary>
+            public static unsafe void StoreVector128x2(byte* address, (Vector128<byte> Value1, Vector128<byte> Value2) value) => StoreVector128x2(address, value);
```

Comment:
Can you also add the following as part of this PR?

```c#
namespace System.Runtime.Intrinsics.Arm
{
    public static class AdvSimd
    {
        public partial class Arm
        {
            // ST2 (multiple structures)
            public static unsafe void StoreVector64x2(byte*   address, (Vector64<byte> Value1,    Vector64<byte> Value2));
            public static unsafe void StoreVector64x2(sbyte*  address, (Vector64<sbyte> Value1,   Vector64<sbyte> Value2));
            public static unsafe void StoreVector64x2(short*  address, (Vector64<short> Value1,   Vector64<short> Value2));
            public static unsafe void StoreVector64x2(ushort* address, (Vector64<ushort> Value1,  Vector64<ushort> Value2));
            public static unsafe void StoreVector64x2(int*    address, (Vector64<int> Value1,     Vector64<int> Value2));
            public static unsafe void StoreVector64x2(uint*   address, (Vector64<uint> Value1,    Vector64<uint> Value2));
            public static unsafe void StoreVector64x2(float*  address, (Vector64<float> Value1,   Vector64<float> Value2));
        }
    }
}
```

Reply:
Sure, that was just a thought because it shouldn't be any different than what you are already doing in this PR except to handle 64 bits in some code.

Reply:
Implemented and pushed with the changes.